cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/SourceControlManagementHttpHandler.java

```diff
@@ -127,6 +126,33 @@ public void deleteRepository(FullHttpRequest request, HttpResponder responder,
                                                               namespaceId));
   }
 
+  /**
+   * Lists applications found in linked repository for the given namespace.
+   * Returns the application name and git-filehash for each application config file found
+   * <pre>
+   * {@code
```

Comment:
nit: Why do you need both `<pre>` and `{@code`?  I don't see special chars that requires `{@code`?

Reply:
removed @code